### DISCUSSION TITLE
Include hashtag(s) when composing post if following

### DISCUSSION THREAD

**Discussion by dcommins** [cheeaun/phanpy#576] [Ideas]
When I'm displaying posts of followed hashtags and I want to compose a post, it would be nice option to include those hashtags in my post. Whether to include or not could be a setting.

**Comment by Fastidious** [comment]
@dcommins did you mean, when replying to a status that has hashtags, you want them to populate automatically on your reply? Or do you want a newly created status to inherit a hashtag filter?

**Reply by dcommins** [reply]
If I'm displaying a list of posts containing one or more hashtags I'm following and I then click the Compose button or even reply to a post from that list, I'd like the option of automatically adding them to my post. The option could be in Settings so I don't have to click through a prompt when I want to compose a new post. 

The inclusion would only be if I'm viewing a list of posts from followed hashtags (i.e. when the URL of the page I'm looking at contains /t/<hashtag(s)>).
